src/subhalo.h

```diff
@@ -114,13 +120,15 @@ class Subhalo_t
     SnapshotIndexOfDeath = SpecialConst::NullSnapshotId;
     SnapshotIndexOfSink = SpecialConst::NullSnapshotId;
     SinkTrackId = SpecialConst::NullTrackId;
-    DescendantTrackId = SpecialConst::NullTrackId;    
+    DescendantTrackId = SpecialConst::NullTrackId;
     MostBoundParticleId = SpecialConst::NullParticleId;
+    TracerIndex = -1;
```

Comment:
Should we change this to SpecialConst::NullParticleId?

Reply:
The tracer index is not a particle ID, so maybe not?

Reply:
True, then it would make it confusing.